src/components/dms/NewChat.tsx

```diff
@@ -253,3 +275,17 @@ function SearchablePeopleList({
     />
   )
 }
+
+function canBeMessaged(profile: AppBskyActorDefs.ProfileView) {
+  if (profile.associated?.chat) {
+    if (profile.associated.chat.allowIncoming === 'all') {
+      return true
+    } else if (
+      profile.associated.chat.allowIncoming === 'following' &&
+      profile.viewer?.followedBy
+    ) {
+      return true
+    }
+  }
+  return false
+}
```

Comment:
Something for the morning: will all users get the default declaration, or do we need to default to true in the client for users without it?

Reply:
Reworked this logic to default to following